roles/local/tasks/main.yml

```diff
@@ -171,6 +171,9 @@
     - svc
   when: process_supervision == 'supervise'
 
+- name: cp tidb-insight tarball
```

Comment:
you could remove this task, and change dest in 'download diagnosis tools' task. https://github.com/pingcap/tidb-ansible/blob/master/roles/local/tasks/main.yml#L87

Reply:
OK